syntax = "proto2";
import "session_event.proto";

message Event_UserMessage {
    extend SessionEvent {
        optional Event_UserMessage ext = 1003;
    }
    optional string sender_name = 1;
    optional string receiver_name = 2;
    optional string message = 3;
}
